src/app/inventory/inventory.component.ts

```diff
@@ -21,9 +21,17 @@ export class InventoryComponent implements OnInit {
   //edit data
   edit = new Item("","","","","",true,true,true);
 
+  setUpDown = "Setup";
+  countAll: number;
```

Comment:
these counts are for testing at the moment. Working to get Deployed, joined, completed, and checkedin counts to display like POC app